water_balance/tools/waterbalance.py

```diff
@@ -818,12 +1045,16 @@ def pop_up_no_agg_found(self):
               "\n- infiltration (in case model has (simple_)infiltration)"\
               "\n\npositive cumulative:\n- discharge"\
               "\n\nnegative cumulative:\n- discharge"
-        QMessageBox.warning(None, header, msg)
+        ms2 = "\n\nNOTE: for 100% closure of the balance, please make sure " \
+              "that the v2_aggregation_settings' timestep for all " \
+              "aggregation variables is a multiple of the v2_global_settings'"\
+              " output_time_step"
+        QMessageBox.warning(None, header, ms1, ms2)
```

Comment:
Waarom is het opgedeeld in 2 messages? Gaat dit goed? Als ik kijk naar de docs lijkt dit niet te kunnen http://pyqt.sourceforge.net/Docs/PyQt4/qmessagebox.html.